draft-ietf-quic-recovery.md

```diff
@@ -1076,8 +1076,11 @@ Pseudocode for OnAckReceived and UpdateRtt follow:
 
 ~~~
 OnAckReceived(ack, pn_space):
-  largest_acked_packet[pn_space] =
-      max(largest_acked_packet[pn_space], ack.largest_acked)
+  if (largest_acked_packet[pn_space] == infinite):
```

Comment:
Would NaN be better?

Reply:
Probably yes.  I'll make a PR for that.

Reply:
I don't know what happened, but somehow this was merged, yet I don't see the relevant changes in the text.  Did another change inadvertently revert this?

Reply:
> Would NaN be better?

I don't think so since NaN != NaN

Reply:
Oh why we didn't we just reserve 0 for uninitialized ...

Reply:
> I don't think so since NaN != NaN

`largest_acked_packet[pn_space] is NaN` (or something similar) would do the trick.

Reply:
`is_uninitialized(largest_acked_packet[pn_space])`

Reply:
Due to an error in my use of Github, you can now have this discussion on #2651